package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tezster",
-  "version": "0.2.3",
+  "version": "0.2.4",
```

Comment:
is 0.2.4 a published version?

Reply:
no, that is the previous version. And in current tezster-dev branch version is mentioned as 0.2.4 in current file as well. I merged my own branch with master start making changes.